### DISCUSSION TITLE
Autofill not working until I click the login button again

### DISCUSSION THREAD

**Discussion by EriksonEnterprises** [livewire/livewire#9003] [1. Help]
I have a login page in livewire 3. Both Email and Password fields are Required on both blade and component.

When I load the login page, my login details are autofilled correctly.

But - if I check the $email and the $password property, they claim empty, even though it's autofilled.

I click login, and nothing happens because the fields are required - so it doesn't login. But, now the $email and $password are not null anymore (from looking at the values echoed out in blade using {{ $email }}).

I click login again, and it logs in.

If I refresh the page, this issue starts all over again

Anyone come across this before?

**Comment by EriksonEnterprises** [comment]
I found some other issues opened about how you have to set the model to model.defer

This seems to work, but I was hoping to have live validation on autofilled data :/

Maybe not possible...

Thank you

**Comment by SlothLady** [comment]
I'm having the same issue with Livewire 3 and adding model.defer doesn't fix it as I don't think this exists in Livewire 3 given the default behaviour of wire:model is supposed to be deferred, the autofilled fields flash for a moment before being emptied while the fields are still yellow from being autofilled.

**Comment by SlothLady** [comment]
Okay if you use wire:model.fill then autocomplete behaves as expected, I think this needs to be made clear in the documentation specifically about autofill behaviour as ".fill 	Use the initial value provided by a "value" HTML attribute on page-load" technically doesn't cover this behaviour.

**Comment by SlothLady** [comment]
PR: #9406
